### PR TITLE
update def rule in markdown.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

File: src/Microsoft.DocAsCode.YamlSerialization/Microsoft.DocAsCode.YamlSerialization.csproj [hunk 1]
```diff
@@ -9,4 +9,8 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="TEMP\" />
+  </ItemGroup>
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.YamlSerialization/Microsoft.DocAsCode.YamlSerialization.csproj]: ?

File: test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs [hunk 2]
```diff
@@ -542,11 +542,8 @@ 2. D
 
   [c]: c",
             @"<ul>
-<li><p>[a]: a</p>
-</li>
-<li><p>[b]: b</p>
-<p>[c]: c</p>
-</li>
+<li></li>
+<li></li>
```
Comment (vicancy) [inline test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs]: github contains only one <li></li>.....?

File: src/Microsoft.DocAsCode.MarkdownLite/Basic/BlockRules/MarkdownDefBlockRule.cs [hunk 3]
```diff
@@ -3,16 +3,59 @@
...
 namespace Microsoft.DocAsCode.MarkdownLite
 {
+    using System;
     using System.Text.RegularExpressions;
 
+    using Microsoft.DocAsCode.MarkdownLite.Matchers;
+
     public class MarkdownDefBlockRule : IMarkdownRule
     {
+        private static readonly Matcher _DefMatcher =
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.MarkdownLite/Basic/BlockRules/MarkdownDefBlockRule.cs]: >D [](start = 41, length = 1)

d

File: src/Microsoft.DocAsCode.YamlSerialization/Microsoft.DocAsCode.YamlSerialization.csproj (hunk 1 above)
Comment (vicancy) [inline src/Microsoft.DocAsCode.YamlSerialization/Microsoft.DocAsCode.YamlSerialization.csproj]: should remove

---
In reply to: [107324542](https://github.com/dotnet/docfx/pull/1455#discussion_r107324542) [](ancestors = 107324542)

File: test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs (hunk 2 above)
Comment (vwxyzh) [inline test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs]: Yes, you can try it in GitHub.

Comment (vwxyzh) [inline test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs]:
```md
* [a]: a

* [b]: b

[c]: c
```
GitHub render it as:
* [a]: a

* [b]: b

[c]: c

**Comment by hellosnow** [comment]
:shipit:

File: test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs (hunk 2 above)
Comment (vicancy) [inline test/Microsoft.DocAsCode.MarkdownLite.Tests/GfmTest.cs]: oops, missing one ` *` when testing --

---
In reply to: [107327425](https://github.com/dotnet/docfx/pull/1455#discussion_r107327425) [](ancestors = 107327425)

**Pull request merged**